script/backup.sh

```diff
@@ -1,52 +1,73 @@
 #!/bin/bash
-if [ "$(id -u)" != "0" ]; then
-  echo "This script must be run as root" 1>&2
+
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /var/tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
   exit 1
 fi
 
-BDIR=$(readlink -f "$1")
-mkdir -p $BDIR
+# Backup dir
+umask 0027
+export BDIR=$1katello-backup-`date "+%Y%m%d%H%M"`
```

Comment:
I don't think we need a date based addition to the specified dir. Lets just use $1 here and use the exact dir the user specifies